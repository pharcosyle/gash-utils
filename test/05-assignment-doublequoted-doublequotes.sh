#set -x
aliaspath=alias
localedir=x
defines="-DALIASPATH=\"${aliaspath}\" -DLOCALEDIR=\"${localedir}\""
echo cc $defines
