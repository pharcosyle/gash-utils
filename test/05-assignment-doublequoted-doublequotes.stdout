cc -DALIASPATH="alias" -DLOCALEDIR="x"
